yt/yql/agent/config.cpp

```diff
@@ -115,6 +136,94 @@ IListNodePtr MergeDefaultSettings(const IListNodePtr& settings, const auto& defa
 
 ////////////////////////////////////////////////////////////////////////////////
 
+void TVanillaJobFile::Register(TRegistrar registrar) {
+    registrar.Parameter("name", &TThis::Name)
+        .NonEmpty();
+    registrar.Parameter("local_path", &TThis::LocalPath)
+        .NonEmpty();
+}
+
+void TDqYtBackend::Register(TRegistrar registrar) {
```

Comment:
nip: do we consider DQ as an acronym or as a name? In former case, it should be DQ not Dq since it's two-letter.

Reply:
Historically, all yql-related code use Dq case.

And I need to use a number of YQL libraries in yql plugin, like this one:
https://github.com/ydb-platform/ydb/pull/1615/files#diff-56940a493503bb073efae9b41ec31d1a67ff6d2849f6d403176b6ec11e2195a9R175

Therefore, I want to make my new code consistent with with the existing one.

Reply:
Okay, makes sense